native/src/main/java/io/ballerina/lib/ldap/ssl/SSLConfig.java

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Shall we use the latest license header. You can find it [here](https://github.com/ballerina-platform/module-ballerina-ldap/blob/f4c5e93bc476d8c750120d0756b8da7985c3da19/native/src/main/java/io/ballerina/lib/ldap/CustomAsyncResultListener.java#L1)

Reply:
Check for other places